test/spell/LinkShareTokenV3Eth.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.28;
+
+import {PoolId} from "src/common/types/PoolId.sol";
+import {ShareClassId} from "src/common/types/ShareClassId.sol";
+
+import {IShareToken} from "src/spoke/interfaces/IShareToken.sol";
+
+import {LinkShareTokenCommon} from "./LinkShareTokenCommon.sol";
+
+/// @notice V3 Ethereum-specific spell that links both JTRSY and JAAA share tokens to V3 system
+/// @dev This spell runs on V3 Guardian/Root and performs V3 operations
+contract LinkShareTokenEth is LinkShareTokenCommon {
+    // See https://www.notion.so/Centrifuge-V3-Initi-Pool-Setup-2322eac24e1780fa84acceaa1ff01dbf
+    PoolId public constant JAAA_POOL_ID = PoolId.wrap(281474976710663);
+    ShareClassId public constant JAAA_SHARE_CLASS_ID = ShareClassId.wrap(0x57e1b211a9ce6306b69a414f274f9998);
```

Comment:
CR

Share class id is not correct. Use `0x00010000000000070000000000000001`

Reply:
Fixed!